.github/ISSUE_TEMPLATE/1-connector-website-not-working.md

```diff
@@ -10,12 +10,16 @@ assignees: ''
 **Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
 Yes/No
 
-**Did you [try Nightly Build](https://github.com/manga-download/hakuneko/releases) ?**
-Yes/No
+**Make sure to [try Nightly Build](https://github.com/manga-download/hakuneko/releases) .**
+Nightly build contains latest fixes, maybe your problem is already solved.
 
-**Is the website of the connector working properly / are you able to see the manga within your browser**
-Yes/No
+**Make sure to [try Nightly Build](https://github.com/manga-download/hakuneko/releases) .**
```

Comment:
do not remove the question about "is the website working", you're creating duplicates
Also i was planning on moving the issues to forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Reply:
Removal was a mistake indeed